build.go

```diff
@@ -30,10 +31,21 @@ func Build(
 ) packit.BuildFunc {
 	return func(context packit.BuildContext) (packit.BuildResult, error) {
 		logger.Title("%s %s", context.BuildpackInfo.Name, context.BuildpackInfo.Version)
+		var projectPath string
+		var err error
 
-		projectPath, err := buildpackYMLParser.ParseProjectPath(filepath.Join(context.WorkingDir, "buildpack.yml"))
-		if err != nil {
-			return packit.BuildResult{}, err
+		projectPath = os.Getenv("BP_DOTNET_PROJECT_PATH")
```

Comment:
I would like to see os.LookupEnv over os.Getenv here. I think that it is a stronger indication of whether or not the env var is set or not.

Reply:
Same goes for the one in detect.

Reply:
Bump

Reply:
What would we do differently given that extra information?